src/Insurello.RabbitMqClient/MqClient.fs

```diff
@@ -13,15 +13,16 @@ module MqClient =
     type private ModelData =
         { channelConsumer: AsyncEventingBasicConsumer
           rpcConsumer: AsyncEventingBasicConsumer
-          pendingRequests: System.Collections.Concurrent.ConcurrentDictionary<string, Result<ReceivedMessage, string> System.Threading.Tasks.TaskCompletionSource> }
+          pendingRequests: System.Collections.Concurrent.ConcurrentDictionary<string, Result<ReceivedMessage, string> System.Threading.Tasks.TaskCompletionSource>
+          connection: IConnection }
```

Comment:
I think you can access the connection from the AsyncEventingBasicConsumer instance

Reply:
Nope